wynnset/tapestry-wp
# 536 - Show a sidebar when selecting a node

Closes #536 

**Changes**

- Adds a sidebar on the side of a Tapestry that shows information on the currently selected node
- Sidebar contains the node title, description, copyright and references information
- Sidebar is responsive and should look good on all screen sizes including mobile
- Reposition node modal so that sidebar is always visible when open

**Testing**

Load a Tapestry, the sidebar should be visible on the right side and span the height of the screen.

- Click on the information icon and the sidebar should slide open with the information icon highlighted green
- Click on the arrow in the middle and the sidebar should close
- Click on the view button should open the lightbox for that node
- Click on the edit button should open the node modal with that node loaded in
  - The node modal should be positioned to the left of the sidebar and not cover it **unless your screen is less than 1280 pixels wide**

Create a node with a very long copyright and reference list (long enough that the sidebar has a scrollbar).

- Select the node, then open the sidebar
- Scroll down. When you reach the copyright title, the green highlight should move to the copyright logo
- Click on the information icon. The sidebar should scroll back up to the node title and description and the information icon should now be highlighted.

Create a node without public edit permissions.

- Open the Tapestry in incognito mode (or log out)
- Click on the node without edit permissions and open the sidebar
- The edit button on the top right corner should not be visible

Shrink your browser so it's <500px wide.

- The sidebar should cover the entire window and the icons should appear at the bottom
